sdcm/nemesis.py

```diff
@@ -1866,6 +1867,34 @@ def modify_table_bloom_filter_fp_chance(self):  # pylint: disable=invalid-name
         """
         self._modify_table_property(name="bloom_filter_fp_chance", val=random.random() / 2)
 
+    def toggle_table_gc_mode(self):
+        """
+            Alters a non-system table tombstone_gc_mode option.
+            Choose the alternate option of 'repair' / 'timeout'
+             (*) The other available values of 'disabled' / 'immediate' are not tested by
+             this nemesis since not applicable to a longevity test.
+        """
+        all_ks_cfs = self.cluster.get_non_system_ks_cf_list(db_node=self.target_node)
+        non_mview_ks_cfs = self.cluster.get_non_system_ks_cf_list(db_node=self.target_node, filter_out_mv=True)
+
+        if not all_ks_cfs:
+            raise UnsupportedNemesis(
+                'Non-system keyspace and table are not found. toggle_table_gc_mode nemesis can\'t run')
+
+        mview_ks_cfs = list(set(all_ks_cfs) - set(non_mview_ks_cfs))
+        keyspace_table = random.choice(all_ks_cfs)
```

Comment:
In case we use seed for SisyphusMonkey, maybe we should reuse this seed here too to make test flow more reproducible?

Reply:
i agree... best is always to have a more controllable way in case we need to reproduce an issue

Reply:
I'm not sure i follow the idea here. The SysyphusMonkey is mainly about reproducing the nemesis order.
As for other random choices, it is right indeed, there's may a room for improvement, perhaps as a separate refactor.
For example, the code now has 77 such random choises as the above one:
```
~/git/scylla-cluster-tests((HEAD detached at upstream/master))$ grep -r "random.choice(" ./* | wc -l
77
```
so if all of this can be controlled by a general seed it's great, or we can concider another refactor for it. is there, btw, an alternative relevant seed to use?

Reply:
we had such situation when working with the reversed queries... as a matter of suggestion, it is always good to have a controllable way to reproduce this issue...
so, at least, what you can do here, is to select a "random seed" and print it to the log, so in case of a failure, we can add in a reproducer this seed to reproduce the exact same scenario we had before.

Reply:
Since there are still some concerns to discuss, i suggest we leave that code as is, and pospone a better solution to a future PR.

Reply:
no, please change to:

```suggestion
        seed = random.randint(1, 100)
        self.log.debug(f'here using {seed=}')
        random.seed(seed)
        keyspace_table = random.choice(all_ks_cfs)
```

Reply:
Why this change wasn't accepted?
Now the randomness here isn't controlled.